src/authorization/dynamic-filters/SitesAdminDynamicAuthorizationFilter.ts

```diff
@@ -21,6 +21,12 @@ export default class SitesAdminDynamicAuthorizationFilter extends DynamicAuthori
         // Override
         authorizationFilters.filters.siteIDs = filteredSiteIDs.filter(
           (siteID) => authorizationFilters.filters.siteIDs.includes(siteID));
+      } else if (Utils.objectHasProperty(extraFilters, 'siteID') &&
```

Comment:
I initially added this because on CREATE & UPDATE SiteArea 'siteID' is being sent instead of 'SiteID'. But I just checked now and with all the changes in place, it is not needed anymore.